### ISSUE TITLE
[Frontend] — Missing Alt Text on Images

### ISSUE THREAD

**Issue by Olowodarey** [Devsol-01/Nestera#623]
**Description:**
Some images lack alt text attributes.

**Expected Behavior:**
Add descriptive alt text to all images for SEO and accessibility.

**Comment by MerlinTheWhiz** [comment]
@MerlinTheWhiz has applied to work on this issue as part of the **Stellar Wave Program**'s 4th wave.


> I'd love to work on this


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/a46221e0-5c78-4370-b742-2a58fe293938) or assign @MerlinTheWhiz to this issue.

**Comment by SYMBAxx** [comment]
@SYMBAxx has applied to work on this issue as part of the **Stellar Wave Program**'s 4th wave.


> can i get assigned to this


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/a46221e0-5c78-4370-b742-2a58fe293938) or assign @SYMBAxx to this issue.
